app/src/main/java/com/sun/noteapp/ui/trash/TrashActivity.kt

```diff
@@ -0,0 +1,208 @@
+package com.sun.noteapp.ui.trash
+
+import android.content.DialogInterface
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import android.view.View
+import android.widget.Toast
+import androidx.appcompat.app.AlertDialog
+import androidx.recyclerview.widget.ItemTouchHelper
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.sun.noteapp.R
+import com.sun.noteapp.data.model.Note
+import com.sun.noteapp.data.repository.NoteLocalRepository
+import com.sun.noteapp.data.source.local.LocalDataSource
+import com.sun.noteapp.data.source.local.NoteDatabase
+import com.sun.noteapp.ui.base.BaseDialog
+import com.sun.noteapp.ui.home.dialog.SortDialog
+import com.sun.noteapp.utils.SWIPE_RTL
+import kotlinx.android.synthetic.main.activity_trash.*
+
+class TrashActivity :
```

Comment:
Parent class nằm cùng dòng.
```suggestion
class TrashActivity : AppCompatActivity(),
```